source/partials/release_notes/_release-19-9-0.md.erb

```diff
@@ -0,0 +1,69 @@
+<h4>Java 8 and 10 support removed</h4>
+
+Support for Java 8 and 10 has been dropped from this release. This is in line with the Java Licensing Changes we had announced earlier this year in this [blog post](https://www.gocd.org/2019/05/21/official-stance-on-java/).
```

Comment:
This is in line with our [official stance on Java](https://www.gocd.org/2019/05/21/official-stance-on-java/) we announced earlier this year post the Java Licensing Changes.

Reply:
done